rc/filetype/just.kak

```diff
@@ -38,18 +38,19 @@ define-command -hidden just-indent-on-new-line %{
 # ‾‾‾‾‾‾‾‾‾‾‾‾
 
 add-highlighter shared/justfile regions
+
 add-highlighter shared/justfile/content default-region group
+add-highlighter shared/justfile/content/recipe regex '^@?([\w-]+)((?:\h[\w-]+(?:=(?:"[^\n]*?"))?)*):(?!=)(\h(?:[\w-]+|\([\w-]+\h[\w-]+\)))?' 1:function 2:value 3:type
+add-highlighter shared/justfile/content/globals regex ^([\w-]+)\h*:=\h*(["|'][^\n]*?["|'])((?:\h*\+\h*[\w-]+)*) 1:meta 2:value 3:meta
+add-highlighter shared/justfile/content/operator regex '((^@|\h:=\h|=(?=")|\h\+\h))' 1:operator
+
 add-highlighter shared/justfile/comment  region '#' '$'  fill comment
-add-highlighter shared/justfile/double_string region '"' (?<!\\)(\\\\)*" fill string
```

Comment:
It's a bit strange that strings are no longer colored as `string` like in other languages, but that's okay.

Reply:
I put them as nested regions into the content region, that works perfeclty and makes a lot of things simpler to paint.